csharp/src/ProtocolBuffers.Test/TestProtos/Unittest.cs

```diff
@@ -2281,6 +2281,19 @@ public sealed partial class Builder : pb::GeneratedBuilder<RepeatedGroup, Builde
     }
     #endregion
 
+    private object oneofField_;
+    public enum OneofFieldOneofCase {
+      OneofUint32 = 111,
+      OneofNestedMessage = 112,
+      OneofString = 113,
+      OneofBytes = 114,
+      OneofFieldNotSet = 0,
```

Comment:
As per previous discussions, None would be more conventional here - we don't need the type name to be a prefix here, as the names only have to be unique within the enum.